src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -2688,6 +2688,8 @@ class MavlinkStreamHomePosition : public MavlinkStream
 					msg.approach_y = 0.0f;
 					msg.approach_z = 0.0f;
 
+					msg.time_usec = hrt_absolute_time();
```

Comment:
Shouldn't this be set to the time the home position was set (home.timestamp) rather than the time it was broadcast over mavlink?

Reply:
I want to change the msg.time_usec = hrt_absolute_time(); to home.timestamp. but i think that there is an error here:
https://github.com/PX4/Firmware/blob/2a7fe22a0408f324ddd8e063323c58479129719b/src/modules/commander/Commander.cpp#L1124-L1131

the update should be done after setting timestamp and manual_home, right?

on other places where home is changed I can see that home.timestamp updtaed correctlly

Reply:
Fixed in https://github.com/PX4/Firmware/pull/11109

Reply:
great